business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -15,7 +15,7 @@
 
   <properties>
     <app.name>business-central</app.name><!-- for tests to know if they are running against business-central-->
-    <selenium.homepage.loading.timeout.seconds>30</selenium.homepage.loading.timeout.seconds>
+    <selenium.homepage.loading.timeout.seconds>45</selenium.homepage.loading.timeout.seconds>
```

Comment:
Hi @jomarko, I guess time out can be event larger (even 90 seconds or more). Since it will wait only in case of need, not always. Thank you!

Reply:
I understand, I propose to wait with updates in this PR, while engineers are solving troubles caused by ip-bom upgrade and then lets see if community selenium tests still fail. WDYT?

Reply:
I am not sure that unstable Selenium tests failing due to ip-bom upgrade. But let's wait, no rushing.

Reply:
Hi @jomarko, so what is the resolution, does it make sense to increase this time out even more?